sym_common.txt

```diff
@@ -98,8 +98,8 @@ gUnknown_3005354: @ 3005354
 	.include "evolution_scene.o"
 	.align 2
 	.include "save.o"
-
 	.align 2
+
```

Comment:
???

Reply:
tried `.include battle_anim_special.o` here but it was giving errors

Reply:
You need to add the common symbols in their vanilla order to `common_syms/battle_anim_special.txt`.